bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperTest.java

```diff
@@ -290,4 +290,16 @@ public void readComplete(int rc, LedgerHandle lh, Enumeration<LedgerEntry> seq,
 
         startBKCluster();
     }
+
+    @Test
```

Comment:
Its good practise to add time out in tests, @Test(timeout = 60000). Typo in test case name, testAutoCloseableBooKeeper.